packages/salesforcedx-vscode-apex-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -791,142 +947,6 @@ function setTypeRefsForEnabledCheckpoints(): boolean {
 // 3. Remove any existing checkpoints
 // 4. Create the new checkpoints
 let creatingCheckpoints = false;
-export async function sfdxCreateCheckpoints(): Promise<boolean> {
```

Comment:
this threw me.  it seems sfdxCreateCheckpoints() was moved.  (why?)  if it hadn't been moved it wouldn't have appeared as a new function above.  I read your comment about a prettier format change, but didn't see code that matched

Reply:
See this [test](https://github.com/forcedotcom/salesforcedx-vscode/pull/4429/files#r976041850).  It's the same issue where a stub wasn't working for an exported function so in order to get around it I moved the function to a static method on the class.  Sorry for the misdirection on the prettier comment I didn't look closely enough on my initial review of the change.